tests/workflows/integration/test-upload-charm/tests/integration/test_charm.py

```diff
@@ -15,7 +15,7 @@ async def test_build_and_deploy(ops_test: OpsTest, pytestconfig):
     """
     app_name = "test"
     assert ops_test.model
-    charm = await ops_test.build_charm(".")
+    charm = pytestconfig.getoption("--charm-file")
```

Comment:
Minor nitpick, should this be better suited in conftest?

Reply:
The other option is also fetched this way. I don't see why not changing it, though